### PR TITLE
Created PMK brand folder including posey_split keyboards with default and via keymaps.

### PR DESCRIPTION
**Pull request by joliverMI**
## Description

-Created a new main keyboard folder for new brand "PMK" (poseysmechanicalkeyboards)
-Created a sub-folder for a keyboard we will sell: posey_split
-Created 2 version folders for two versions we have in circulation (version 4 and 5):
Included "default" and "via" keymaps for each version.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by joliverMI** [comment]
@zvecr Thanks for promptly reviewing my PR. This is my first PR. I did review the PR checklist several times, so any violations are surely from my lack of understanding. If you could point me to the general areas that need improvement, I'd really appreciate it.

File: keyboards/pmk/posey_split/posey_split_v4/keymaps/VIA/keymap.c
Comment (waffle87) [inline keyboards/pmk/posey_split/posey_split_v4/keymaps/VIA/keymap.c:1]: This directory must be renamed to lowercase.

File: keyboards/pmk/posey_split/posey_split_v4/keymaps/default/config.h
Comment (waffle87) [inline keyboards/pmk/posey_split/posey_split_v4/keymaps/default/config.h:1]: This file should be removed.

File: keyboards/pmk/posey_split/posey_split_v4/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,73 @@
+// Copyright 2024 Javier Oliver (@joliverMI)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+
+//System Layers
+#define _QWERTY 0
+#define _NAV 8
+#define _SYMB 9
```
Comment (waffle87) [inline keyboards/pmk/posey_split/posey_split_v4/keymaps/default/keymap.c:10]:
```suggestion
enum layers {
    _QWERTY,
    _NAV,
    _SYMB
};
```

File: keyboards/pmk/posey_split/posey_split_v4/keymaps/VIA/keymap.c [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+     //├─────────────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┼────────┴───────┤
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_ENT,
+     //├─────────────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┴────────────────┤
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_UP,
+     //├──────────┬──────┴─┬───────┼───────┴────────┼────────┴────────┴───────┤ ├────────┴────────┴────────┼────────┼────────┼────────┼────────────────┤
+        KC_TRNS,   KC_TRNS, KC_TRNS,  KC_TRNS,                 KC_TRNS,                   KC_TRNS,          KC_TRNS, KC_TRNS, KC_LEFT, KC_DOWN, KC_RIGHT
+     //└──────────┴────────┴───────┴────────────────┴─────────────────────────┘ └──────────────────────────┴────────┴────────┴────────┴───────┴────────┘
+    ),
+    [_SYMB] = LAYOUT_game_split_85(
```
Comment (waffle87) [inline keyboards/pmk/posey_split/posey_split_v4/keymaps/VIA/keymap.c:58]:
```suggestion
    [_QWERTY] = LAYOUT(
    // ┌────────┐        ┌────────┬────────┬────────┬────────┬────────┬───────┐ ┌────────┬────────┬────────┬────────┬────────┬────────┬────────┬───────┐
        KC_ESC,  KC_NO,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,    KC_F6,   MO(_SYMB),  KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,   KC_F13,
     //├────────┼────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┼────────┴───────┤
        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,     KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,   KC_BACKSPACE,
     //├────────┴────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┼────────┬───────┤
        KC_TAB,           KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,     KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
     //├─────────────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┼────────┴───────┤
        KC_CAPS,          KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,     KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,      KC_ENT,
     //├─────────────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┼────────────────┤
        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,     KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,   KC_UP,
     //├──────────┬──────┴─┬───────┼───────┴────────┼────────┴────────┴───────┤ ├────────┴────────┴────────┼────────┼────────┼────────┼────────────────┤
        KC_LCTL,   KC_LGUI, MO(_NAV),    KC_LALT,                  KC_SPC,                    KC_SPC,          MO(_NAV),   KC_RGUI, KC_LEFT, KC_DOWN,  KC_RIGHT
     //└──────────┴────────┴───────┴────────────────┴─────────────────────────┘ └──────────────────────────┴────────┴────────┴────────┴───────┴────────┘
    ),
    [_NAV] = LAYOUT(
    // ┌────────┐        ┌────────┬────────┬────────┬────────┬────────┬───────┐ ┌────────┬────────┬────────┬────────┬────────┬────────┬────────┬───────┐
        KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,   KC_TRNS, KC_TRNS, MO(_SYMB), KC_TRNS, KC_VOLD, KC_VOLU, KC_MUTE, KC_MPRV, KC_MPLY, KC_MNXT,
     //├────────┼────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┼────────┴───────┤
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
     //├────────┴────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┼────────┬───────┤
        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
     //├─────────────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┼────────┴───────┤
        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_ENT,
     //├─────────────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┴────────────────┤
        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_UP,
     //├──────────┬──────┴─┬───────┼───────┴────────┼────────┴────────┴───────┤ ├────────┴────────┴────────┼────────┼────────┼────────┼────────────────┤
        KC_TRNS,   KC_TRNS, KC_TRNS,  KC_TRNS,                 KC_TRNS,                   KC_TRNS,          KC_TRNS, KC_TRNS, KC_LEFT, KC_DOWN, KC_RIGHT
     //└──────────┴────────┴───────┴────────────────┴─────────────────────────┘ └──────────────────────────┴────────┴────────┴────────┴───────┴────────┘
    ),
    [_SYMB] = LAYOUT(
```

File: keyboards/pmk/posey_split/posey_split_v4/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+     //├─────────────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┼────────┴───────┤
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_ENT,
+     //├─────────────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┴────────────────┤
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_UP,
+     //├──────────┬──────┴─┬───────┼───────┴────────┼────────┴────────┴───────┤ ├────────┴────────┴────────┼────────┼────────┼────────┼────────────────┤
+        KC_TRNS,   KC_TRNS, KC_TRNS,  KC_TRNS,                 KC_TRNS,                   KC_TRNS,          KC_TRNS, KC_TRNS, KC_LEFT, KC_DOWN, KC_RIGHT
+     //└──────────┴────────┴───────┴────────────────┴─────────────────────────┘ └──────────────────────────┴────────┴────────┴────────┴───────┴────────┘
+    ),
+    [_SYMB] = LAYOUT_game_split_85(
```
Comment (waffle87) [inline keyboards/pmk/posey_split/posey_split_v4/keymaps/default/keymap.c:58]:
```suggestion
    [_QWERTY] = LAYOUT(
    // ┌────────┐        ┌────────┬────────┬────────┬────────┬────────┬───────┐ ┌────────┬────────┬────────┬────────┬────────┬────────┬────────┬───────┐
        KC_ESC,  KC_NO,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,    KC_F6,   MO(_SYMB),  KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,   KC_F13,
     //├────────┼────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┼────────┴───────┤
        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,     KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,   KC_BACKSPACE,
     //├────────┴────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┼────────┬───────┤
        KC_TAB,           KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,     KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
     //├─────────────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┼────────┴───────┤
        KC_CAPS,          KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,     KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,      KC_ENT,
     //├─────────────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┼────────────────┤
        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,     KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,   KC_UP,
     //├──────────┬──────┴─┬───────┼───────┴────────┼────────┴────────┴───────┤ ├────────┴────────┴────────┼────────┼────────┼────────┼────────────────┤
        KC_LCTL,   KC_LGUI, MO(_NAV),    KC_LALT,                  KC_SPC,                    KC_SPC,          MO(_NAV),   KC_RGUI, KC_LEFT, KC_DOWN,  KC_RIGHT
     //└──────────┴────────┴───────┴────────────────┴─────────────────────────┘ └──────────────────────────┴────────┴────────┴────────┴───────┴────────┘
    ),
    [_NAV] = LAYOUT(
    // ┌────────┐        ┌────────┬────────┬────────┬────────┬────────┬───────┐ ┌────────┬────────┬────────┬────────┬────────┬────────┬────────┬───────┐
        KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,   KC_TRNS, KC_TRNS, MO(_SYMB), KC_TRNS, KC_VOLD, KC_VOLU, KC_MUTE, KC_MPRV, KC_MPLY, KC_MNXT,
     //├────────┼────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┼────────┴───────┤
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
     //├────────┴────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┼────────┬───────┤
        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
     //├─────────────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┼────────┴───────┤
        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_ENT,
     //├─────────────────┼────────┼────────┼────────┼────────┼────────┼───────┤ ├────────┼────────┼────────┼────────┼────────┼────────┴────────────────┤
        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_UP,
     //├──────────┬──────┴─┬───────┼───────┴────────┼────────┴────────┴───────┤ ├────────┴────────┴────────┼────────┼────────┼────────┼────────────────┤
        KC_TRNS,   KC_TRNS, KC_TRNS,  KC_TRNS,                 KC_TRNS,                   KC_TRNS,          KC_TRNS, KC_TRNS, KC_LEFT, KC_DOWN, KC_RIGHT
     //└──────────┴────────┴───────┴────────────────┴─────────────────────────┘ └──────────────────────────┴────────┴────────┴────────┴───────┴────────┘
    ),
    [_SYMB] = LAYOUT(
```

File: keyboards/pmk/posey_split/posey_split_v4/posey_split_v4.c
Comment (waffle87) [inline keyboards/pmk/posey_split/posey_split_v4/posey_split_v4.c:1]: This file should be removed.

File: keyboards/pmk/posey_split/posey_split_v4/posey_split_v4.h [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    { k23, k24, k25, k26, k27, k28, k29, kno}, \
+    { k30, kno, k31, k32, k33, k34, k35, k36}, \
+    { k37, k38, k39, k40, kno, kno, k41, kno}, \
+    { k42, k43, k44, k45, k46, k47, k48, k49}, \
+    { k50, k51, k52, k53, k54, k55, k56, kno}, \
+    { k57, k58, k59, k60, k61, k62, k63, k64}, \
+    { k65, k66, k67, k68, k69, k70, k71, kno}, \
+    { k72, k73, k74, k75, k76, k77, k78, kno}, \
+    { kno, k79, k80, k81, k82, k83, kno, k84} \
+}
```
Comment (waffle87) [inline keyboards/pmk/posey_split/posey_split_v4/posey_split_v4.h:30]: This must migrated to `info.json` and this file removed.

File: keyboards/pmk/posey_split/posey_split_v4/readme.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+* Hardware Supported: posey_split_v4 and logic_module_v1
+* Hardware Availability: poseysmechanicalkeyboards.com
+
+Make example for this keyboard (after setting up your build environment):
+
+    make posey_split_v4:default
+
+Flashing example for this keyboard:
+
+    make posey_split_v4:default:flash
```
Comment (waffle87) [inline keyboards/pmk/posey_split/posey_split_v4/readme.md:17]:
```suggestion
![posey_split_v4](https://i.imgur.com/g3yf6alh.png)


* Keyboard Maintainer: [Javier Oliver](https://github.com/joliverMI)
* Hardware Supported: posey_split_v4 and logic_module_v1
* Hardware Availability: poseysmechanicalkeyboards.com

Make example for this keyboard (after setting up your build environment):

    make pmk/posey_split/posey_split_v4:default
    
Flashing example for this keyboard:

    make pmk/posey_split/posey_split_v4:default:flash
```

File: keyboards/pmk/posey_split/posey_split_v4/rules.mk [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+# RGB Configuration
+WS2812_DRIVER = vendor
+RGBLIGHT_ENABLE = yes
+RGBLIGHT_DRIVER = ws2812 
+
+# Allow String Macros
+SEND_STRING_ENABLE = yes
+ 
+# Split Board Configuration
+SERIAL_DRIVER = usart
```
Comment (waffle87) [inline keyboards/pmk/posey_split/posey_split_v4/rules.mk:12]:
```suggestion
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Comment by waffle87** [comment]
Please apply the same changes to the `posey_split_v5` directory as well. Mind you, what is the difference between the two revision?

File: keyboards/pmk/posey_split/posey_split_v4/config.h [hunk 7]
```diff
@@ -0,0 +1,28 @@
+// Copyright 2023 Javier Oliver (@joliverMI)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/*Split keyboard configuration*/
+// Hardware Settings
+#define SPLIT_HAND_PIN GP18 //GP18 at High is for Left
+#define SERIAL_PIO_USE_PIO1
```
Comment (zvecr) [inline keyboards/pmk/posey_split/posey_split_v4/config.h:9]:
```suggestion
```

File: keyboards/pmk/posey_split/posey_split_v4/info.json [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+            "enabled": true,
+            "max": 4
+        },
+        "led_count": 1,
+        "sleep": true,
+        "split": true,
+        "split_count": [1, 0]
+    },
+    "split": {
+        "enabled": true,
```
Comment (zvecr) [inline keyboards/pmk/posey_split/posey_split_v4/info.json:38]:
```suggestion
        "enabled": true,
        "handedness": {
            "pin": "GP18"
        }
```

File: keyboards/pmk/posey_split/posey_split_v4/info.json [hunk 9]
```diff
@@ -0,0 +1,147 @@
+{
+    "manufacturer": "Posey's Mechanical Keyboards",
+    "keyboard_name": "posey_split_v4",
+    "maintainer": "joliverMI",
+    "bootloader": "rp2040",
+    "build": {
+        "debounce_type": "sym_defer_pk"
+    },
+    "debounce": 5,
```
Comment (zvecr) [inline keyboards/pmk/posey_split/posey_split_v4/info.json:9]:
```suggestion
```

File: keyboards/pmk/posey_split/posey_split_v4/config.h [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+// Communication Settings
+#define FORCED_SYNC_THROTTLE_MS 1000
+#define SPLIT_MAX_CONNECTION_ERRORS 5
+#define SPLIT_CONNECTION_CHECK_TIMEOUT 2000
+
+// RGB Configuration
+#define WS2812_DI_PIN GP17
+#define RGBLIGHT_EFFECT_BREATHING
+#define WS2812_PIO_USE_PIO1
+#define NOP_FUDGE 0.4
```
[the remark this answers is not in the document]
Comment (zvecr) [reply keyboards/pmk/posey_split/posey_split_v4/config.h:24]: `WS2812_PIO_USE_PIO1` wouldnt really be required either.
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]
As with the previous reviews, the same should be applied to the other keyboard.

Also, the folder structure would be better as
```
keyboards/pmk/posey_split/posey_split/v4/info.json
keyboards/pmk/posey_split/posey_split/v4/rules.mk
keyboards/pmk/posey_split/posey_split/v5/info.json
keyboards/pmk/posey_split/posey_split/v5/rules.mk
```

With potentially common items pulled to the new parent folder.

**Comment by joliverMI** [comment]
> As with the previous reviews, the same should be applied to the other keyboard.
> 
> Also, the folder structure would be better as
> 
> ```
> keyboards/pmk/posey_split/posey_split/v4/info.json
> keyboards/pmk/posey_split/posey_split/v4/rules.mk
> keyboards/pmk/posey_split/posey_split/v5/info.json
> keyboards/pmk/posey_split/posey_split/v5/rules.mk
> ```
> 
> With potentially common config pulled to the new parent folder.

**Comment by joliverMI** [comment]
@waffle87 Thank you (and the other reviewers) for so much detailed help. I'm new to all this but I learned a lot from this process and really appreciate your time. I think I got everything, and I've tested the new setup and it seems to be working. Is there a process to put this PR back in review?

File: keyboards/pmk/posey_split/v4/info.json [hunk 11]
```diff
@@ -0,0 +1,151 @@
...
+        "enabled": true,
+        "handedness": {
+            "pin": "GP18"
+        }
+        "matrix_pins": {
+            "right": {
+                "cols": ["GP2", "GP3", "GP4", "GP5", "GP6", "GP7", "GP8", "GP9"],
+                "rows": ["GP10", "GP11", "GP12", "GP13", "GP14", "GP15"]
+            }
+        }
```
Comment (zvecr) [inline keyboards/pmk/posey_split/v4/info.json:51]:
```suggestion
```

File: keyboards/pmk/posey_split/v4/keymaps/VIA/keymap.c
Comment (zvecr) [inline keyboards/pmk/posey_split/v4/keymaps/VIA/keymap.c:1]: The `VIA` folder still needs to be renamed to lower case

File: keyboards/pmk/posey_split/v5/readme.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
+# posey_split_v5
+
+![posey_split_v5](https://i.imgur.com/g3yf6al.png)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Javier Oliver](https://github.com/joliverMI)
+* Hardware Supported: *The PCBs, controllers supported*
+* Hardware Availability: *Links to where you can find this hardware*
```
Comment (zvecr) [inline keyboards/pmk/posey_split/v5/readme.md:9]: This still needs filling out

File: keyboards/pmk/posey_split/v5/rules.mk [hunk 13]
```diff
@@ -0,0 +1,2 @@
+# Split Board Configuration
```
Comment (zvecr) [inline keyboards/pmk/posey_split/v5/rules.mk:1]:
```suggestion
```

File: keyboards/pmk/posey_split/v5/keymaps/VIA/keymap.c [hunk 14]
```diff
@@ -0,0 +1,73 @@
+// Copyright 2024 Javier Oliver (@joliverMI)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+
+//System Layers
+#define _QWERTY 0
+#define _NAV 8
+#define _SYMB 9
```
Comment (zvecr) [inline keyboards/pmk/posey_split/v5/keymaps/VIA/keymap.c:10]:
```suggestion
enum my_layers {
    _QWERTY,
    _NAV,
    _SYMB,
};
```

File: keyboards/pmk/posey_split/v4/keymaps/default/keymap.c [hunk 15]
```diff
@@ -0,0 +1,73 @@
+// Copyright 2024 Javier Oliver (@joliverMI)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+
+//System Layers
+#define _QWERTY 0
+#define _NAV 8
+#define _SYMB 9
```
Comment (zvecr) [inline keyboards/pmk/posey_split/v4/keymaps/default/keymap.c:10]:
```suggestion
enum my_layers {
    _QWERTY,
    _NAV,
    _SYMB,
};
```

File: keyboards/pmk/posey_split/v4/keymaps/VIA/keymap.c [hunk 16]
```diff
@@ -0,0 +1,73 @@
+// Copyright 2024 Javier Oliver (@joliverMI)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+
+//System Layers
+#define _QWERTY 0
+#define _NAV 8
+#define _SYMB 9
```
Comment (zvecr) [inline keyboards/pmk/posey_split/v4/keymaps/VIA/keymap.c:10]:
```suggestion
enum my_layers {
    _QWERTY,
    _NAV,
    _SYMB,
};
```

**Review by zvecr — CHANGES_REQUESTED** [review]
Many requested changes have not been completed.

**Comment by joliverMI** [comment]
> Many requested changes have not been completed.
My apologies, I certainly did miss a few. I hope I haven't missed more this time.

I noticed the PR Lint and the QMK CI Build checks failed. Is there something I can learn from that? Does it point to an issue with my code?

Thanks so much

File: keyboards/pmk/posey_split/v4/readme.md [hunk 17]
```diff
@@ -0,0 +1,34 @@
+# posey_split_v4
+
+![posey_split_v4](https://i.imgur.com/g3yf6alh.png)
+
+Posey's Split Mechanical Keyboards Highlights
+
+*A short description of the keyboard/project*
+About Posey's Split Mechanical Keyboards
```
Comment (zvecr) [inline keyboards/pmk/posey_split/v4/readme.md:8]:
```suggestion
```

File: keyboards/pmk/posey_split/v5/readme.md [hunk 18]
```diff
@@ -0,0 +1,32 @@
+# posey_split_v5
+
+![posey_split_v5](https://i.imgur.com/g3yf6al.png)
+
+*A short description of the keyboard/project*
+About Posey's Split Mechanical Keyboards
```
Comment (zvecr) [inline keyboards/pmk/posey_split/v5/readme.md:6]:
```suggestion
```

File: keyboards/pmk/posey_split/v4/keymaps/via/keymap.c [hunk 19]
```diff
@@ -0,0 +1,73 @@
+// Copyright 2024 Javier Oliver (@joliverMI)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+
+//System Layers
+#define _QWERTY 0
+#define _NAV 8
+#define _SYMB 9
```
Comment (zvecr) [inline keyboards/pmk/posey_split/v4/keymaps/via/keymap.c:10]: This has been requested multiple times
```suggestion
enum my_layers {
    _QWERTY,
    _NAV,
    _SYMB,
};
```

File: keyboards/pmk/posey_split/rules.mk
Comment (zvecr) [inline keyboards/pmk/posey_split/rules.mk:1]: This file should be removed.

**Review by zvecr — CHANGES_REQUESTED** [review]

**Comment by joliverMI** [comment]
Hello, I just wanted to check and make sure I've made necessary changes and that I am just waiting for review?

**Review by waffle87 — APPROVED** [review]

**Pull request merged**